lib/views/profile/profile.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/material.dart';
+import 'package:thc/models/enum_widget.dart';
+import 'package:thc/models/navigator.dart';
+import 'package:thc/views/profile/edit_profile.dart';
+import 'package:thc/views/profile/heart_center_info.dart';
+import 'package:thc/views/profile/issue_report.dart';
+import 'package:thc/views/profile/settings.dart';
+
+enum ProfileOption with StatelessEnum {
+  edit(
+    icon: Icons.edit,
+    label: 'edit profile',
+    page: EditProfile(),
+  ),
+  settings(
+    icon: Icons.settings,
+    label: 'settings',
+    page: SettingsScreen(),
+  ),
+  info(
+    icon: Icons.info_outline,
+    label: 'about The Heart Center',
+    page: HeartCenterInfo(),
+  ),
+  report(
+    icon: Icons.report_problem,
+    label: 'report an issue',
+    page: IssueReport(),
+  ),
+  logout(
+    icon: Icons.logout,
+    label: 'sign out',
+  );
```

Comment:
I really liked Hongbo's idea of showing a circular profile pic, so I decided to build off of that.

I'm very pleased with how this looks, both the code and the UI.

![comparison](https://github.com/theheartcenter-one/thc/assets/10457200/eaebcd3a-bee2-4781-9f79-12f527354b76)